apps/docs/src/guide/errors/index.md

```diff
@@ -0,0 +1,3 @@
+# Errors
+
+<!-- Will replace this file's contents with the content from PR #1866 once it's merged -->
```

Comment:
Don't merge before #1866 since both this PR and #1866 are adding this new file, but #1866 contains the contents that we actually want for this specific file